src/dart-2.md

```diff
@@ -2,23 +2,25 @@
 layout: default
 permalink: /dart-2
 title: Dart 2 Updates
-description: How Dart 2 is different from Dart 1.x, and how you can prepare.
+description: How Dart 2 is different from Dart 1.x, and how you can convert your code to work with Dart 2.
 ---
 
-This page has information about changes that are coming in Dart 2,
-and how you can migrate your code from Dart 1.x.
+This page has information about changes in Dart 2,
+and how you can migrate your code from Dart 1.x and
+older 2.0.0-dev releases.
 
 <aside class="alert alert-warning" markdown="1">
-**Don't rely on Dart 2 until it's stable.**
-Dart 2 pre-releases will have a series of incompatible changes.
-Although we appreciate your willingness to prepare for and test Dart 2,
-please don't bet your livelihood on a pre-release.
-For information on potential changes, see the
+**Dart 2 is still changing.**
```

Comment:
"still changing" sounds a bit too scary IMO. What Dart 2 is is not changing. Maybe "Dart 2 is still landing"?